include/zapierwrapper.js

```diff
@@ -2,4 +2,10 @@
 const path = require('path');
 const zapier = require('zapier-platform-core');
 const appPath = path.resolve(__dirname, 'index.js');
-module.exports = {handler: zapier.createAppHandler(appPath)};
+let opts;
+try {
+  opts = require(appPath).appFlags;
```

Comment:
Can we call it just `flags` or `options`? Having `app` in the name feels redundant.